src/Nethermind/Nethermind.TxPool/TxBroadcaster.cs

```diff
@@ -291,7 +296,13 @@ private void NotifyPeersAboutLocalTx(Transaction tx)
 
         public bool TryGetPersistentTx(Keccak hash, out Transaction? transaction)
         {
-            return _persistentTxs.TryGetValue(hash, out transaction);
+            if (_persistentTxs.TryGetValue(hash, out transaction) && !transaction.SupportsBlobs)
+            {
+                return true;
+            }
+
+            transaction = default;
+            return false;
```

Comment:
ok can we simplify:

```
            bool found = _persistentTxs.TryGetValue(hash, out transaction);
            return found && !transaction!.SupportsBlobs;
```

Reply:
Then it will return `false` and light blob tx - right now it is fine as we use it only in one place and we check `true`/`false`, but it would be possible to use it wrong way in the future. Your simplification looks better, but initial implementation is more safe - returns `null` instead of light blob tx